crates/chaotic_semantic_memory_duckdb/README.md

```
@@ -14,11 +14,41 @@ let batch = analytics.query("SELECT count(*) FROM concepts")?;
+top_connected = (
+    associations
+    .group_by("src_id")
+    .count()
+    .sort("count", descending=True)
+    .limit(10)
+    .join(concepts, left_on="src_id", right_on="id")
+)
+print(top_connected)
+```
 
 ## Usage
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)